pkg/cli/aws/aws.go

```diff
@@ -51,6 +51,25 @@ func DefaultCredentialOptions() *CredentialOptions {
 	}
 }
 
+// DefaultCredentialOptionsWithLog uses the default role and profile
+// for accessing AWS.
+func DefaultCredentialOptionsWithLog(log logrus.FieldLogger) *CredentialOptions {
+	b, err := box.LoadBox()
+	if err != nil {
+		return nil
+	}
+
+	if log == nil {
+		log = logrus.New()
+	}
```

Comment:
suggestion: I think `log` should be an explicit parameter. Otherwise you should document that it's optional.
```suggestion
```